### PR TITLE
API-8253: default value in lc3 props

### PR DESCRIPTION
**Pull request by czeczot**
## 🚀 Links

- [[Feature branch]](https://developers.labs.livechat.com/docs/feature/API-8253-default_value_in_property_config)
- [[Jira]](https://livechatinc.atlassian.net/browse/API-8185) 

## ✅ Checklist (for API docs)

- Changelog
- API versions
- Web & RTM
- **Table:** Available methods/webhooks/pushes
- **Table:** Specifics

For more guidelines, see [Updating API docs](https://livechatinc.atlassian.net/wiki/spaces/PAT/pages/761529400/Updating+API+docs).

### Content

- New content
- Proofreading/content fixes
  
### Features (for the website)

- New feature
- Bug fix
- Breaking change

### CODE REVIEW

File: content/management/configuration-api/v3.3/index.mdx [hunk 1]
```diff
@@ -1695,6 +1695,11 @@ Property domain can be configured in two ways:
 - by defining a set of values explicitly allowed in this property (for example `[1, 2, 3]`).
 - by defining a range. All values within the range are allowed in this property. It works only for numeric types (for example a range from `1` to `3`).
 
+## Property default value
+
+If the value of a property has not been set explicitly then **default value** will be returned, if any.
```
Comment (ale-kacperczyk) [inline content/management/configuration-api/v3.3/index.mdx:1700]:
```suggestion
If a property value hasn't been explicitly set, then the **default value** will be returned, if any.
```

File: content/management/configuration-api/v3.3/index.mdx [hunk 2]
```diff
@@ -1695,6 +1695,11 @@ Property domain can be configured in two ways:
...
+The **default value** of the property, can be set during property registration.
```
Comment (ale-kacperczyk) [inline content/management/configuration-api/v3.3/index.mdx:1701]:
```suggestion
The **default value** of a property can be set during [property registration](#register-property) as the `deafult_value` parameter.
```

File: content/management/configuration-api/v3.3/index.mdx [hunk 3]
```diff
@@ -1794,6 +1799,7 @@ Registers a new private property for a given **Client Id**.
 | `range` __*__              | No                  | `object`   | Range of values that properties can be set to                                         |
 | `range.from`               | No                  | `int`      | Only values **equal** or **greater** than this parameter can be set to this property  |
 | `range.to`                 | No                  | `int`      | Only values **equal** or **lower** than this parameter can be set to this property    |
+| `default_value`            | No                  | `type`     | Default value of property, validated by **domain** or **range** if one exists         |
```
Comment (ale-kacperczyk) [inline content/management/configuration-api/v3.3/index.mdx:1802]:
```suggestion
| `default_value`            | No                  | `type`     | Default value of a property; validated by **domain** or **range**, if one exists.         |
```

File: content/management/configuration-api/v3.3/index.mdx [hunk 4]
```diff
@@ -2049,6 +2056,7 @@ curl -X POST \
 ### List License Properties
 
 Returns the properties set within a license.
+For properties without explicit property value assignment, will return default values if exists.
```
Comment (ale-kacperczyk) [inline content/management/configuration-api/v3.3/index.mdx:2059]:
```suggestion
For properties without an explicit value assignment, the method will return the default values (if they exist).
```

File: content/management/configuration-api/v3.3/index.mdx [hunk 5]
```diff
@@ -2215,6 +2223,7 @@ curl -X POST \
 ### List Group Properties
 
 Returns the properties set within a group.
+For properties without explicit property value assignment, will return default values if exists.
```
Comment (ale-kacperczyk) [inline content/management/configuration-api/v3.3/index.mdx:2226]: The same as above

**Review by ale-kacperczyk — CHANGES_REQUESTED** [review]

File: content/management/configuration-api/v3.3/index.mdx (hunk 3 above)
Comment (zuczkows) [reply content/management/configuration-api/v3.3/index.mdx:1802]: Maybe it's worth mentioning that it's only possible for group and license (property locations) 🤔 ?

**Review by litwinow — APPROVED** [review]

File: content/management/configuration-api/v3.3/index.mdx [hunk 6]
```diff
@@ -1794,8 +1799,10 @@ Registers a new private property for a given **Client Id**.
 | `range` __*__              | No                  | `object`   | Range of values that properties can be set to                                         |
 | `range.from`               | No                  | `int`      | Only values **equal** or **greater** than this parameter can be set to this property  |
 | `range.to`                 | No                  | `int`      | Only values **equal** or **lower** than this parameter can be set to this property    |
+| `default_value` __**__     | No                  | `type`     | Default value of property; validated by **domain** or **range**, if one exists        |
 
 __*)__ Only one `domain` and one `range` can be set for a single property.
+__**)__ Register is possible only if `access.<location>` set to `license` or `group`.
```
Comment (ale-kacperczyk) [inline content/management/configuration-api/v3.3/index.mdx:1805]:
```suggestion
__**)__ Registering a property is possible only if `access.<location>` is set to `license` or to `group`.
```

File: content/management/configuration-api/v3.3/index.mdx [hunk 7]
```diff
@@ -2247,7 +2257,7 @@ curl -X POST \
   -H 'Authorization: Bearer <your_access_token>' \
   -d '{
       "id": 1,
-      "namespace_prefix": "0",
+      "namespace": "0",
```
Comment (ale-kacperczyk) [inline content/management/configuration-api/v3.3/index.mdx:2260]:
```suggestion
      "namespace": "0805e283233042b37f460ed8fbf22160",
```

Comment (ale-kacperczyk) [reply content/management/configuration-api/v3.3/index.mdx:2260]: namespace "0" wasn't a good example

File: content/management/configuration-api/v3.3/index.mdx [hunk 8]
```diff
@@ -2079,7 +2088,7 @@ curl -X POST \
   -H 'Content-Type: application/json' \
   -H 'Authorization: Bearer <your_access_token>' \
   -d '{
-      "namespace_prefix": "0",
+      "namespace": "0",
```
Comment (ale-kacperczyk) [inline content/management/configuration-api/v3.3/index.mdx:2091]:
```suggestion
      "namespace": "0805e283233042b37f460ed8fbf22160",
```

File: content/management/configuration-api/v3.3/index.mdx (hunk 6 above)
Comment (ale-kacperczyk) [reply content/management/configuration-api/v3.3/index.mdx:1805]: So if you want to access a property from chat you need to register it for license/group + chat?

Comment (czeczot) [reply content/management/configuration-api/v3.3/index.mdx]: registering chat / thread / event property is not possible with default value

File: content/management/configuration-api/v3.3/index.mdx [hunk 9]
```diff
@@ -1695,6 +1695,11 @@ Property domain can be configured in two ways:
 - by defining a set of values explicitly allowed in this property (for example `[1, 2, 3]`).
 - by defining a range. All values within the range are allowed in this property. It works only for numeric types (for example a range from `1` to `3`).
 
+## Property default value
+
+If a property value hasn't been explicitly set, then the **default value** will be returned, if any.
+The **default value** of a property can be set during [property registration](#register-property) as the `deafult_value` parameter.
```
Comment (ale-kacperczyk) [inline content/management/configuration-api/v3.3/index.mdx:1701]:
```suggestion
The **default value** of a property can be set during [property registration](#register-property) as the `default_value` parameter.
```

File: content/management/configuration-api/v3.3/index.mdx (hunk 6 above)
Comment (ale-kacperczyk) [reply content/management/configuration-api/v3.3/index.mdx]: ok, ok, I forgot it's only for a property with default value :D

**Review by ale-kacperczyk — APPROVED** [review]

File: content/management/configuration-api/v3.3/index.mdx (hunk 6 above)
Comment (czeczot) [reply content/management/configuration-api/v3.3/index.mdx]: yes, my description was little bit wrong but i fixed it now should be more clear :)

**Pull request merged**